dlt/common/destination/reference.py

```diff
@@ -139,6 +139,11 @@ def update_storage_schema(self, staging: bool = False, only_tables: Iterable[str
         self._verify_schema()
         return expected_update
 
+    @abstractmethod
+    def drop_tables(self, *tables: str, staging: bool = False) -> None:
```

Comment:
this should go to `SqlJobClientBase`. `JobClientBase` may not support any "tables" (ie. be an event sink or a parquet file). you need to test it as wel

Reply:
Make sense. Done. Test added in https://github.com/dlt-hub/dlt/pull/285/commits/87be84bf8758f9aa75b9b44890795372cb38d5c5